pages/portal.tsx

```diff
@@ -212,7 +212,7 @@ const Subdomain = () => {
                 >
                   <PrimaryButton
                     title={'Create My Portal'}
-                    linkTo={'https://portal.fileverse.io/#/claim'}
+                    linkTo={'https://portal.fileverse.io/#/create?code=ANN'}
```

Comment:
can you revert the link to claim link? Its disabled from backend.

Reply:
Okay